modin/pandas/__init__.py

```diff
@@ -134,6 +134,7 @@ def _update_engine(publisher: Parameter):
                 from distributed import Client
 
                 dask_client = Client(n_workers=CpuCount.get())
+                num_cpus = len(dask_client.cluster.plan)
```

Comment:
shouldn't this be done outside of `except`? inside `except` there is the logic where we override the cpu cound (and we can get cpu count from `CpuCount.get()`)

Reply:
Definitely, thanks!